Excel chromatogram export fails

<img width="1920" height="1165" alt="Image" src="https://github.com/user-attachments/assets/b1ae5f98-c93b-440d-83e1-5572c111e96e" />

```
java.lang.reflect.InvocationTargetException
	at org.eclipse.jface.operation.ModalContext.run(ModalContext.java:395)
	at org.eclipse.jface.dialogs.ProgressMonitorDialog.run(ProgressMonitorDialog.java:470)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.processChromatogram(ExtendedChromatogramUI.java:545)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.executeSupplier(ExtendedChromatogramUI.java:756)
	at org.eclipse.chemclipse.ux.extension.ui.editors.ProcessorSupplierMenuEntry.execute(ProcessorSupplierMenuEntry.java:96)
	at org.eclipse.swtchart.extensions.core.ScrollableChart$6.widgetSelected(ScrollableChart.java:1641)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:286)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:91)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:5884)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1656)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:5099)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4540)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1151)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1042)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:153)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:678)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:339)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:583)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:173)
	at org.eclipse.chemclipse.rcp.app.ui.internal.support.ApplicationSupportDefault.start(ApplicationSupportDefault.java:26)
	at org.eclipse.chemclipse.rcp.app.ui.Application.start(Application.java:28)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:219)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:149)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:115)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:467)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:298)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:627)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:575)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1431)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1403)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.apache.poi.xssf.model.SharedStringsTable
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(Unknown Source)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.apache.poi.xssf.usermodel.XSSFFactory.createDocumentPart(XSSFFactory.java:56)
	at org.apache.poi.ooxml.POIXMLFactory.newDocumentPart(POIXMLFactory.java:109)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:588)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:498)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.onWorkbookCreate(XSSFWorkbook.java:468)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:259)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:253)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:241)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.io.ChromatogramWriter.writeChromatogram(ChromatogramWriter.java:63)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.converter.ChromatogramExportConverter.convert(ChromatogramExportConverter.java:45)
	at org.eclipse.chemclipse.converter.chromatogram.AbstractChromatogramConverter.convert(AbstractChromatogramConverter.java:288)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:82)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:1)
	at org.eclipse.chemclipse.model.supplier.ChromatogramSelectionProcessSupplier.apply(ChromatogramSelectionProcessSupplier.java:43)
	at org.eclipse.chemclipse.model.supplier.IChromatogramSelectionProcessSupplier$1.execute(IChromatogramSelectionProcessSupplier.java:56)
	at org.eclipse.chemclipse.processing.supplier.IProcessSupplier.applyProcessor(IProcessSupplier.java:151)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$9(ExtendedChromatogramUI.java:759)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.executeMethod(ExtendedChromatogramUI.java:1816)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$8(ExtendedChromatogramUI.java:756)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:123)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchMethodError: 'void org.apache.xmlbeans.XmlOptions.put(java.lang.Object)' [in thread "ModalContext"]
	at org.apache.poi.xssf.model.SharedStringsTable.<clinit>(SharedStringsTable.java:94)
	... 28 more
Root exception:
java.lang.NoClassDefFoundError: Could not initialize class org.apache.poi.xssf.model.SharedStringsTable
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(Unknown Source)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.apache.poi.xssf.usermodel.XSSFFactory.createDocumentPart(XSSFFactory.java:56)
	at org.apache.poi.ooxml.POIXMLFactory.newDocumentPart(POIXMLFactory.java:109)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:588)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:498)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.onWorkbookCreate(XSSFWorkbook.java:468)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:259)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:253)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:241)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.io.ChromatogramWriter.writeChromatogram(ChromatogramWriter.java:63)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.converter.ChromatogramExportConverter.convert(ChromatogramExportConverter.java:45)
	at org.eclipse.chemclipse.converter.chromatogram.AbstractChromatogramConverter.convert(AbstractChromatogramConverter.java:288)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:82)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:1)
	at org.eclipse.chemclipse.model.supplier.ChromatogramSelectionProcessSupplier.apply(ChromatogramSelectionProcessSupplier.java:43)
	at org.eclipse.chemclipse.model.supplier.IChromatogramSelectionProcessSupplier$1.execute(IChromatogramSelectionProcessSupplier.java:56)
	at org.eclipse.chemclipse.processing.supplier.IProcessSupplier.applyProcessor(IProcessSupplier.java:151)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$9(ExtendedChromatogramUI.java:759)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.executeMethod(ExtendedChromatogramUI.java:1816)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$8(ExtendedChromatogramUI.java:756)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:123)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchMethodError: 'void org.apache.xmlbeans.XmlOptions.put(java.lang.Object)' [in thread "ModalContext"]
	at org.apache.poi.xssf.model.SharedStringsTable.<clinit>(SharedStringsTable.java:94)
	... 28 more
```

and 

```
java.lang.NoClassDefFoundError: Could not initialize class org.apache.poi.xssf.model.SharedStringsTable
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(Unknown Source)
	at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(Unknown Source)
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.apache.poi.xssf.usermodel.XSSFFactory.createDocumentPart(XSSFFactory.java:56)
	at org.apache.poi.ooxml.POIXMLFactory.newDocumentPart(POIXMLFactory.java:109)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:588)
	at org.apache.poi.ooxml.POIXMLDocumentPart.createRelationship(POIXMLDocumentPart.java:498)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.onWorkbookCreate(XSSFWorkbook.java:468)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:259)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:253)
	at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:241)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.io.ChromatogramWriter.writeChromatogram(ChromatogramWriter.java:63)
	at org.eclipse.chemclipse.msd.converter.supplier.excel.converter.ChromatogramExportConverter.convert(ChromatogramExportConverter.java:45)
	at org.eclipse.chemclipse.converter.chromatogram.AbstractChromatogramConverter.convert(AbstractChromatogramConverter.java:288)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:82)
	at org.eclipse.chemclipse.msd.converter.chromatogram.ChromatogramConverterMSDProcessTypeSupplier$ChromatogramConverterMSDProcessorSupplier.apply(ChromatogramConverterMSDProcessTypeSupplier.java:1)
	at org.eclipse.chemclipse.model.supplier.ChromatogramSelectionProcessSupplier.apply(ChromatogramSelectionProcessSupplier.java:43)
	at org.eclipse.chemclipse.model.supplier.IChromatogramSelectionProcessSupplier$1.execute(IChromatogramSelectionProcessSupplier.java:56)
	at org.eclipse.chemclipse.processing.supplier.IProcessSupplier.applyProcessor(IProcessSupplier.java:151)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$9(ExtendedChromatogramUI.java:759)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.executeMethod(ExtendedChromatogramUI.java:1816)
	at org.eclipse.chemclipse.ux.extension.xxd.ui.swt.editors.ExtendedChromatogramUI.lambda$8(ExtendedChromatogramUI.java:756)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:123)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchMethodError: 'void org.apache.xmlbeans.XmlOptions.put(java.lang.Object)' [in thread "ModalContext"]
	at org.apache.poi.xssf.model.SharedStringsTable.<clinit>(SharedStringsTable.java:94)
	... 28 more


```

--- a/openchrom/releng/net.openchrom.targetplatform/net.openchrom.targetplatform.target
+++ b/openchrom/releng/net.openchrom.targetplatform/net.openchrom.targetplatform.target
@@ -35,6 +35,7 @@
 		<unit id="org.apache.poi"/>
 		<unit id="org.apache.poi.ooxml" />
 		<unit id="org.apache.poi.ooxml.schemas" />
+		<unit id="org.apache.xmlbeans"/>
 	</location>
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 		<repository location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/release/4.37.0"/>

